sites/public/package.json

```diff
@@ -53,6 +53,7 @@
     "@babel/core": "^7.11.6",
     "@cypress/code-coverage": "^3.9.12",
     "@cypress/webpack-preprocessor": "^5.11.1",
+    "@netlify/plugin-nextjs": "4.30.4",
```

Comment:
It does look like this is bringing in a lot of subdependencies we don't need, is the goal to have a followup that tries to remove this or leave it as is?

Reply:
Yeah, I'm hoping this will no longer be needed when we migrate to next 13. I'll add a comment to that issue to be investigated at that time